src/executables/loaddb.c

```diff
@@ -99,8 +99,8 @@ static int schema_file_start_line = 1;
 static int index_file_start_line = 1;
 static int compare_Storage_order = 0;
 
-#define LOADDB_LOG_FILENAME "loaddb.log"
-static FILE *loaddb_log_file;
+#define LOADDB_LOG_FILENAME_SUFFIX "loaddb.log"
+FILE *loaddb_log_file = NULL;
```

Comment:
What is the reason for stripping `static`? It indicates the scope of global variable and protects you from declaring it `extern` someplace else.